bindings/python/spatial/expose-explog.cpp

```diff
@@ -15,22 +15,22 @@ namespace pinocchio
     void exposeExplog()
     {
       
-      bp::def("exp3",&exp3<Eigen::Vector3d>,
+      bp::def("exp3",&exp3<Eigen::VectorXd>,
               bp::arg("Angular velocity (vector of size 3)"),
               "Exp: so3 -> SO3. Return the integral of the input"
               " angular velocity during time 1.");
       
-      bp::def("Jexp3",&Jexp3_proxy<Eigen::Vector3d>,
+      bp::def("Jexp3",&Jexp3_proxy<Eigen::VectorXd>,
```

Comment:
Why are you doing that: changing from Vector3d to VectorXd?

Reply:
That's what I explain above: it was giving wrong results with `Vector3d,` so I had to switch to `VectorXd`

Reply:
It is really weird. It is better to investigate first eigenpy rather than putting some fix here.

Reply:
I would rather have a working system now and think about solving this problem later. The transition on Pinocchio 2.0 is already tough enough. The correction as it stands now does not do a lot of harm, and it will be very easy to roll back.
Anyhow, you can take the newly-added lines in unittest/python/explog as a reference.
In this PR I also include a correction to python's log function.